tr/community/ruby-core/writing-patches/index.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+title: "Yama Yazıcı Kılavuzu"
+lang: tr
+---
+
+Burada yamalarınızın göz önüne alınabilmesi için takip etmeniz gereken, Matz
+tarafından hazırlanmış, bazı ipuçlarını bulabilirsiniz.
+{: .summary}
+
+Bu kılavuz Ruby-Core e-posta listesindeki
+[Matz'in bir gönderisi][ruby-core-post]nden alınmıştır:
+
+* Yama başına bir değişiklik gerçekleyin
+
+  Bu, ertelenen yamaların çoğundaki en büyük meseledir. Birden fazla hatayı
```

Comment:
`Ertelenen yamalar için bu durum en büyük konudur.` gibi bisi olsa daha anlasilir olur sanki

Reply:
Dediğin gibi yapıp gönderdim.